### DISCUSSION TITLE
3DMGAME论坛 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#112] [模板讨论]
3DMGAME论坛 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by oumoe** [comment]
日志：Failed at 4/11 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 404: Not Found,
Request URL: https://bbs.3dmgame.com/thread-{{tid}}-1-1.html
